tesseract_command_language/include/tesseract_command_language/profile_dictionary.h

```diff
@@ -191,11 +159,50 @@ class ProfileDictionary
   std::shared_ptr<const ProfileType> getProfile(const std::string& ns, const std::string& profile_name) const
   {
     std::shared_lock lock(mutex_);
-    const auto& it = profiles_.at(ns);
-    const auto& it2 = it.at(std::type_index(typeid(ProfileType)));
+
```

Comment:
I am not sure how I feel about try,catch in the motion planning pipeline because it a huge time penalty when thrown an caught. Is there a reason to catch an rethrow and not let the exception flow up?

Reply:
I think we should let the this flow up and not catch an rethrow.

Reply:
Note: This will be addressed in profile refactor